rules/android/android_binary.bzl

```
@@ -94,3 +95,11 @@ def android_binary(
         name = name + ".lint",
         target = name,
     )
+
+    android_lint_test(
+        name = name + "_lint_test",
+        srcs = attrs.get("srcs", default = []),
+        resource_files = resources_xml(resource_files),
+        lib = name,
+        deps = android_binary_deps,
```

We need to provide the manifest in order for lint to recognise it as an Android lint
```suggestion
        deps = android_binary_deps,
        manifest = attrs.get("manifest", default = None),
```